### ISSUE TITLE
Clicking towards any clickable link while on Watch Page of Video renders 404 Error

### ISSUE THREAD

**Issue by bivanalhar** [Coursemology/coursemology2#7434]
When user is inside the page `/courses/{courseId}/videos/{videoId}/submissions/{videoSubmissionId}/edit`, clicking any clickable entity (whether it's sidebar icon, or link in Breadcrumbs) will render 404 error.

## Possible Cause

Upon investigating, when we click any entity that redirect the page to any other page, the App will firstly visit the page  `/courses/{courseId}/videos/null/submissions/null/edit`, which obviously will render the 404 Error page since there should not be any video with ID equals to null.

Even though in production, this error is ignored, but it's quite annoying and there's no telling if in any further upgrade, this will still be ignored by the system.

Basically, other than that link, system also makes attempts to visit the real page, which gets prioritised and thus still got redirected to the right page

**Comment by cysjonathan** [comment]
Cannot be reproduced, likely resolved in breadcrumbs PR.
